streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -718,14 +719,11 @@ public void run() {
         try {
             runLoop();
             cleanRun = true;
-        } catch (final KafkaException e) {
-            log.error("Encountered the following unexpected Kafka exception during processing, " +
-                          "this usually indicate Streams internal errors:", e);
-            throw e;
         } catch (final Exception e) {
             // we have caught all Kafka related exceptions, and other runtime exceptions
             // should be due to user application errors
-            log.error("Encountered the following error during processing:", e);
+            log.error("Encountered the following exception during processing " +
+                "and the thread is going to shutdown: ", e);
```

Comment:
```suggestion
                "and the thread is going to shut down: ", e);
```

I'm a nerd, I know... "shutdown" is a noun, "shut down" is a verb phrase ("down" modifies the verb "shut").